main.go

```diff
@@ -137,6 +141,10 @@ func init() {
 	lsAppsCmd.Flags().StringVar(&appSpaceFlag, "space", "", "specify the application space")
 
 	fixerCmd.Flags().StringSliceVar(&fixerSpacesFlag, "spaces", nil, "Specify spaces")
+	oldVersionsCmd.Flags().StringVar(&appSpaceFlag, "space", "", "specify the application space")
+	oldVersionsCmd.Flags().IntVar(&minorFlag, "minor", 2, "specify the maximum number of major versions to expire")
```

Comment:
s/to expire/to keep/

If we have 5 versions, and this parameter is 2, we will throw 3 expired versions and keep 2 versions, right?